AdminWebsite/AdminWebsite.UnitTests/Mappers/NotificationMappers/MapToDemoOrTestNotificationTests.cs

```diff
@@ -17,11 +19,13 @@ public void Should_map_ejud_judge_demo_or_test_notification()
             {
                 Id = Guid.NewGuid(),
                 Participants = new List<ParticipantResponse>(),
-                ScheduledDateTime = new DateTime(2020, 2, 10, 12, 15, 0, DateTimeKind.Utc)
+                ScheduledDateTime = new DateTime(2020, 2, 10, 12, 15, 0, DateTimeKind.Utc),
+                OtherInformation = new OtherInformationDetails { JudgeEmail = "judge@hmcts.net", JudgePhone = "123456789" }.ToOtherInformationString()
             };
             const NotificationType expectedNotificationType = NotificationType.EJudJudgeDemoOrTest;
             const string testType = "Generic";
             const string caseNumber = "MBFY/17364";
+            const string expectedJudgeEmail = "judge@hmcts.net";
```

Comment:
This is the same as the email string on line 23. Could this be defined before that and used to set JudgeEmail there?